requirements-dev.txt

```diff
@@ -50,5 +50,3 @@ h5py
 nni
 optuna
 git+https://github.com/Project-MONAI/MetricsReloaded@monai-support#egg=MetricsReloaded
-nnunetv2 # nnunetv2
```

Comment:
can't set the requirement here because it'll requires torch>1.12 everywhere https://github.com/MIC-DKFZ/nnUNet/blob/1a95bfa0a1483e3a57da55d612a0f914f5ddbef4/setup.py#L12